mkt/games/views.py

```diff
@@ -0,0 +1,23 @@
+from mkt.fireplace.serializers import (FireplaceESAppSerializer,
+                                       FireplaceESWebsiteSerializer)
+from mkt.games.filters import DailyGamesFilter
+from mkt.games.paginator import ESGameAggregationPaginator
+from mkt.search.serializers import DynamicSearchSerializer
+from mkt.search.views import MultiSearchView
+
+
+class DailyGamesView(MultiSearchView):
+    authentication_classes = []
+    cors_allowed_methods = ['get']
+    filter_backends = [DailyGamesFilter]
```

Comment:
You want at least `PublicAppsFilter` too, probably `DeviceTypeFilter` as well, right ? Otherwise you are going to return non-public or incompatible content